### PR TITLE
Move mock events to new demo plugin

### PR DESCRIPTION
**Pull request by bleonard**

### CODE REVIEW

File: plugins/@grouparoo/demo/package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+  "author": "Grouparoo Inc <hello@grouparoo.com>",
+  "name": "@grouparoo/demo",
+  "description": "The Grouparoo Demo Setup Plugin",
+  "version": "0.1.12-alpha.1",
+  "license": "MPL-2.0",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
```
Comment (evantahler) [inline plugins/@grouparoo/demo/package.json:10]: Maybe we don't actually want to publish this one?  
Lerna knows not to publish things that are `private` (defined here)

**Review by evantahler — COMMENTED** [review]
Legit!

File: plugins/@grouparoo/demo/package.json (hunk 1 above)
Comment (bleonard) [reply plugins/@grouparoo/demo/package.json:10]: maybe. I didn't do it on purpose per se, but I can imagine a world where someone wants to load it up with data. doesn't seem to hurt.

Comment (bleonard) [reply plugins/@grouparoo/demo/package.json:10]: whichever you think.

Comment (evantahler) [reply plugins/@grouparoo/demo/package.json:10]: I'm indifferent at the moment - just calling out the options we have.   It'll get published tonight!

**Pull request merged**